docs/API/V1/upload.md

```diff
@@ -0,0 +1,101 @@
+# `PUT /api/v1/upload/<file>`
+
+This API creates a dataset resource by uploading a performance tarball.
```

Comment:
"a performance tarball" sounds like one which is sort of teardrop shaped.  😆 

Should that be "a performance result tarball"?  Or, "workload measurement tarball"?  Or, "benchmark result tarball"?

Reply:
So, maybe "Pbench teardrop(TM)" to make Pbench sound sleek and cool??